src/pages/page4/page4.html

```diff
@@ -39,24 +30,20 @@
 
     <ion-item>
       <ion-label>Sound</ion-label>
-      <ion-select [(ngModel)]="sound" name="sound">
-        <ion-option value="silent">Silent</ion-option>
-        <ion-option value="beep">Beep</ion-option>
-        <ion-option value="ring">Ring</ion-option>
-        <ion-option value="alarm">Alarm</ion-option>
-      </ion-select>
+      <ion-toggle [(ngModel)] = "sound_toggle" name="sound_toggle" value="sound_toggle" (ionChange)="toggleSound()" [disabled]="!notification_toggle"></ion-toggle>
```

Comment:
The toggles don't seem to lag me in the emulator, and it runs fine on my phone. So it should be ok to leave them enabled.